include/aikido/planner/parabolic/ParabolicTimer.hpp

```diff
@@ -30,17 +30,18 @@ namespace parabolic {
 /// \param _maxAcceleration maximum acceleration for each dimension
 /// \return time optimal trajectory that satisfies acceleration constraints
 std::unique_ptr<aikido::trajectory::Spline> computeParabolicTiming(
-      const aikido::trajectory::Spline& _inputTrajectory,
-      const Eigen::VectorXd& _maxVelocity,
-      const Eigen::VectorXd& _maxAcceleration);
+    const aikido::trajectory::Spline& _inputTrajectory,
+    const Eigen::VectorXd& _maxVelocity,
+    const Eigen::VectorXd& _maxAcceleration);
 
 /// Convert an interpolated trajectory to a spline trajectory
-/// This function requires the \c _inputTrajectory to use a \c GeodesicInterpolator.
+/// This function requires the \c _inputTrajectory to use a \c
+/// GeodesicInterpolator.
```

Comment:
Nit: I thought "So the conversion ..." was the continuing sentence without line breaking.